rhamt/entities/projects.py

```diff
@@ -14,43 +14,24 @@
 from rhamt.base.application.implementations.web_ui import ViaWebUI
 from rhamt.base.modeling import BaseCollection
 from rhamt.base.modeling import BaseEntity
-from rhamt.entities import BaseLoggedInPage
+from rhamt.entities import AllProjectView
 from rhamt.entities.analysis_results import AnalysisResultsView
 from rhamt.utils import conf
 from rhamt.utils.ftp import FTPClientWrapper
 from rhamt.utils.update import Updateable
-from rhamt.widgetastic import ProjectList
+from rhamt.widgetastic import DropdownMenu
 from rhamt.widgetastic import ProjectSteps
 from rhamt.widgetastic import TransformationPath
 
 
-class AllProjectView(BaseLoggedInPage):
```

Comment:
Q: why we are moving project view to `__init__.py`. is it not possible to direct use it?

Reply:
It is possible but all other pages have to first go to AllProjectView and then go to their accordions so all the entities will be using this view . I thought of putting it in __init__.py as the common place to import

Reply:
**Optional:** I think we have project.py in view so better to place in same file.